pr_agent/algo/utils.py

```diff
@@ -365,15 +376,6 @@ def try_fix_yaml(response_text: str) -> dict:
             return data
         except:
             pass
```

Comment:
**Suggestion:** In the `try_fix_yaml` function, the `except` clause is empty. This can make debugging difficult because it swallows all exceptions. Consider logging the exception or re-raising it after handling. [best practice]